src/adhocracy_core/docs/deletion.rst

```diff
@@ -1,123 +1,123 @@
 
-  Often the last modification will have been the deletion or hiding of
-  the resource, but there is no guarantee that this is always the case.
-  Especially, the resource may be marked as hidden/deleted because one of its
-  ancestors was hidden/deleted (as that status is inherited). In that case,
-  the person who last modified the child resource likely has nothing to do
-  with the person who hid/deleted the ancestor resource.
-* *Deleted* or *hidden* resources may still be referenced from other
-  resources. If the frontend follows such references it must therefore
-  be prepared to encounter *410 Gone* responses and deal with them
-  appropriately (e.g. by silently skipping them or by showing an
-  explanation such as "Comment deleted").
+  Often the last modification will have been the hiding of the resource,
+  but there is no guarantee that this is always the case.  Especially,
+  the resource may be marked as hidden because one of its ancestors was
+  hidden (as that status is inherited). In that case, the person who
+  last modified the child resource likely has nothing to do with the
+  person who hid the ancestor resource.
+* *Hidden* resources may still be referenced from other resources. If
+  the frontend follows such references it must therefore be prepared to
+  encounter *410 Gone* responses and deal with them appropriately (e.g.
+  by silently skipping them or by showing an explanation such as
+  "Comment deleted").
 * FIXME Not implemented yet, since the frontend doesn't yet need it.
-  *Deleted* and *hidden* resources will normally not be shown in
-  backreferences, which are calculated on demand. The
-  *include=deleted|hidden|all* parameter can be used to change that and
-  include backreferences to deleted and/or hidden resources. The same
-  restrictions apply, i.e. normal users can use this parameter to find out
-  whether hidden backreferences exist, but they won't be able to see their
-  contents. In any case the frontend should be prepared to deal with
-  *410 Gone* when following backreferences in the same way as when
-  following forward reference -- even if it didn't explicitly ask to include
-  them, they might show up due to caching.
+  *Hidden* resources will normally not be shown in backreferences, which
+  are calculated on demand. The *include=hidden* parameter can be used
+  to change that and include backreferences to hidden resources. The
+  same restrictions apply, i.e. normal users can use this parameter to
+  find out whether hidden backreferences exist, but they won't be able
+  to see their contents. In any case the frontend should be prepared to
+  deal with *410 Gone* when following backreferences in the same way as
+  when following forward reference -- even if it didn't explicitly ask
+  to include them, they might show up due to caching.
 
 FIXME We should extend the Meta API to expose the distinction between
 references and backreferences to the frontend, currently only the backend
```

Comment:
cleanup not related to this pull request: can be removed to cleanup fixmes we won't fix or are part of the refactoring list